src/components/form.scss

```diff
@@ -13,89 +13,89 @@ label {
 }
 
 .form__label {
-  padding-bottom: var(--rms-space-x-small);
+  padding-bottom: var(--rm-space-x-small);
   margin-bottom: 0;
   line-height: 1.5;
-  color: var(--rms-colors-on-background);
+  color: var(--rm-color-on-background);
   font-weight: 400;
-  font-size: var(--rms-font-small);
+  font-size: var(--rm-font-small);
   letter-spacing: 0.4px;
 }
 
 .form__input {
   height: auto;
   display: block;
   width: 100%;
-  padding: var(--rms-space-small) var(--rms-space-medium);
-  font-size: var(--rms-font-small);
+  padding: var(--rm-space-small) var(--rm-space-medium);
+  font-size: var(--rm-font-small);
   line-height: 1.5;
```

Comment:
@Jeremy-Walton Can we use our tokens here instead of the static styles for line-height.